README.md

```diff
@@ -9,12 +9,22 @@ further details.
 
 ## Compatibility
 
-- Ember Bootstrap v5
-- Ember CP Validations v5
+- Ember Bootstrap v5 or above
+- Ember CP Validations v5 or above
 - Ember.js v3.28 or above
 - Ember CLI v3.28 or above
 - Node.js v20 or above
 
+## Requirements
```

Comment:
Why do you feel this chapter is needed? It seems to be a duplication of the above.